Assets/MixedRealityToolkit.Services/InputSystem/MixedRealityInputModule.cs

```diff
@@ -0,0 +1,317 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.EventSystems;
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    [RequireComponent(typeof(Camera))]
+    public class MixedRealityInputModule : StandaloneInputModule, IMixedRealityPointerHandler, IMixedRealitySourceStateHandler
+    {
+#if UNITY_EDITOR
+        [UnityEditor.CustomEditor(typeof(MixedRealityInputModule))]
+        public class Editor : UnityEditor.Editor
+        {
+            public override void OnInspectorGUI()
+            {
+                MixedRealityInputModule t = (MixedRealityInputModule)target;
```

Comment:
>t [](start = 40, length = 1)

inputModule (or inputTarget) (I think anything that isn't a single-letter variable name)